docs/releasenotes/release-1.131.md

```diff
@@ -0,0 +1,14 @@
+## Announcement    
+
+## New Beta Resources (Direct Reconciler):
+
+* [`IAPSettings`](https://cloud.google.com/config-connector/docs/reference/resource-docs/iap/iapsettings)
```

Comment:
I tried to open https://cloud.google.com/config-connector/docs/reference/resource-docs/iap/iapsettings and it's 404 not found at the moment. I guess we should add the link after it's published?

Reply:
I think the page is going to be generated by the release process?
I am just following the old template.

Reply:
Yes, it'll be generated possibly as the last step of the release. I think we need to at least ensure the actual release note (not here) contains working URLs. Wondering if GH has any checker for that.